src/fixup.js

```diff
@@ -217,6 +217,12 @@
         }
       }
 
+      window.onclick = function(event) {
```

Comment:
Probably want to use a listener here, as we don't know if dependent documents have set up their own `window.onclick =`
 
```suggestion
      window.addEventListener("click", function(event) {
```